docs/source/index.rst

```diff
@@ -22,6 +22,7 @@ Contents
    content
    history
    batching
+   addons
```

Comment:
FYI: If you want to supress the Sphinx warning, but don't want it included in the toctree yet, you can add `:orphan:` at the top of the file.